src/components/EditEventForm/index.jsx

```diff
@@ -24,26 +23,37 @@ const months = [
   'December',
 ];
 
-let days = [];
-for (let i = 1; i <= 31; i++) {
+const days = [];
+for (let i = 1; i <= 31; i += 1) {
   days.push(i);
 }
 
-let hours = [];
-for (let i = 1; i <= 12; i++) {
+const hours = [];
+for (let i = 1; i <= 12; i += 1) {
   hours.push(i);
 }
 
-const EditEventForm = props => {
+const EditEventForm = (props) => {
```

Comment:
ngl we needa put this in a utils.js or something but we'll refactor later

Reply:
I agree. I didn't want to rearrange code more than it required and risk breaking anything.